packages/commerce-sdk-react/src/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import CommerceAPIProvider from './provider'
+import useCommerceApi from './hooks/useCommerceApi'
+export * from './hooks/types'
+export * from './hooks'
+
+export {CommerceAPIProvider, useCommerceApi}
```

Comment:
When you run `npm run build` in this package, do you see a duplicate nested `dist/dist` folder? I'm seeing that, and the nested folder has a single file called 'index.js'.

Reply:
I see a duplicate `dist`. We need to fix it, but it's a minor issue since that doesn't affect the build. I'm going to defer fixing that later, outside of this PR.